File: typescript/day0/day0.ts
import { readLinesInFile } from "../utils.ts";

function FizzBuzz(n: number): string[] {
  console.log(readLinesInFile("./day0/input.txt"));
  const ans = [];
  for (let i = 1; i <= n; i++) {
    let val = "";
    if (i % 3 === 0) {
      val += "Fizz";
    }
    if (i % 5 === 0) {
      val += "Buzz";
    }
    if (val === "") {
      val = String(i);
    }
    ans.push(val);
  }
  return ans;
}

export { FizzBuzz };

File: python/advent_of_code_2022/advent_of_code_2022/day0/day0.py
# Test problem in preparation for advent of code 2022

def fizz_buzz(n: int):
    ans = []
    for i in range(1, n + 1):
        val = ""
        if i % 3 == 0:
            val += "Fizz"
        if i % 5 == 0:
            val += "Buzz"
        if not val:
            val = str(i)
        ans.append(val)
    return ans
        